lighthouse-core/gather/gatherers/manifest.js

```diff
@@ -26,35 +26,28 @@ const manifestParser = require('../../lib/manifest-parser');
  * the manifest parser.
  */
 class Manifest extends Gatherer {
-
-  static _errorManifest(errorString) {
-    return {
-      raw: undefined,
-      value: undefined,
-      debugString: errorString
-    };
-  }
-
+  /**
+   * Returns the parsed manifest or null, if the page had no manifest.
+   * @param {!Object} options
+   * @return {!Promise<?Manifest>}
+   */
   afterPass(options) {
     return options.driver.sendCommand('Page.getAppManifest')
       .then(response => {
         // We're not reading `response.errors` however it may contain critical and noncritical
```

Comment:
> wdyt about handling these?

do we have a bug tracking it? I assume you added that comment :)

> it might help us differentiate between bad JSON, missing, or other issues.

the first two should be handled, but yeah, other issues would be good to catch